### ISSUE TITLE
Build Software base ( Back end )

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#32]
## User story
1. As a software developer
2. I want / need a software base built respecting the architecture
3. So that features can easily be added

## Acceptance criteria
* Choose and implement the database driver
* Create a dockerfile
* Define an architecture

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
